src/topp/ConsensusID.cpp

```diff
@@ -373,6 +380,31 @@ class TOPPConsensusID :
       new_sp.setMetaValue(SE+":precursor_mass_tolerance_unit",sp.precursor_mass_tolerance_ppm  ? "ppm" : "Da");
       new_sp.setMetaValue(SE+":digestion_enzyme",sp.digestion_enzyme.getName());
       new_sp.setMetaValue(SE+":enzyme_term_specificity",EnzymaticDigestion::NamesOfSpecificity[sp.enzyme_term_specificity]);
+      
+      const auto& chg_pair = sp.getChargeRange();
+      if (chg_pair.first != 0 && chg_pair.first < min_chg) min_chg = chg_pair.first;
+      if (chg_pair.second != 0 && chg_pair.second > max_chg) max_chg = chg_pair.second;
+      if (sp.missed_cleavages > mc ) mc = sp.missed_cleavages;
+      if (sp.fragment_mass_tolerance > frag_tol) frag_tol = sp.fragment_mass_tolerance;
```

Comment:
do you check if unit is the same?

Reply:
no, because then it cannot be combined. there is actually more to check, also for database and enzyme etc.

Reply:
Suggestions welcome. It just has to be independent of the order. I currently assume e.g. same or similar enzymes, which I probably have to change.
Kind of a release blocker, since this blocks the PLFQ pipeline.